### PR TITLE
Modified ReactNative.md file to provide introduction on Expo

### PR DESCRIPTION
**Pull request by zhen-s**
I'v modified the existing `ReactNative.md` page to include more details on React Native and Expo, and formatted the page into a more readable style.

- Added introduction to React Native and Expo so new starters can understand what these tech stacks mean
- Added various links at suitable positions within the description so readers can easily access the official documentations if they require additional information
- Modified the existing style of page and included some more accurate details
- Incorporated my experiences from managing our project so future students can maintain a clean project and avoid conflicts or redundancies.
- Added various hyperlinks to other pages within the wiki at appropriate texts. Reduces overlap with existing info on the wiki while allowing readers to find more details on various concepts, this adds cohesiveness to the entire learning centre.

### CODE REVIEW

File: Topics/Tech_Stacks/ReactNative.md [hunk 1]
```diff
@@ -1,30 +1,51 @@
...
+
+With Expo's [EAS Update](https://docs.expo.dev/eas-update/introduction/) and [EAS Build](https://docs.expo.dev/build/introduction/) features, development builds can be launched instantly, and your app can be tested on mobile devices or mobile device simulators. Official builds can also be launched efficiently to share among your team, or even make its way to AppStore and Google Play. Additionally, Expo offers a [web dashboard](https://expo.dev) with concise UI that aids the management of projects and their deployment. You can sign up for a free Expo account and manage your projects on this dashboard. The [Expo-CLI](https://docs.expo.dev/more/expo-cli/) is the command-line tool that connects all these tools together, and even helps to securely manage your project's dependencies.<br>
```
Comment (HermesCui) [inline Topics/Tech_Stacks/ReactNative.md:11]: SunZhen expanded on the explanation of React Native, including more comprehensive resources for beginners learning React Native. Additionally, he introduced Expo, highlighting its importance as a tool for mobile device testing.

File: Topics/Tech_Stacks/ReactNative.md [hunk 2]
```diff
@@ -1,30 +1,51 @@
...
-         2. Install expo go on your ios device (https://apps.apple.com/us/app/expo-go/id982107779) or android device (https://play.google.com/store/apps/details?id=host.exp.exponent&hl=en_CA&gl=US&pli=1)
+            #Note: Suggested to download the LTS version (or even numbered versions npm) because it is generally more stable. Use `npm -v` and `Node -v` to check your installed version.
+         2. Install Expo Go on your [iOS device](https://apps.apple.com/us/app/expo-go/id982107779) or [Android device](https://play.google.com/store/apps/details?id=host.exp.exponent&hl=en_CA&gl=US&pli=1) via App Store and Google Play Store
          3. Download VS Code Extensions
             1. ES7+ React/Redux/React-Native snippets (Extension ID: dsznajder.es7-react-js-snippets)
             2. React Native Tools (Extension ID: msjsdiag.vscode-react-native)
       2. Command
-         1. Install expo
+         1. Install Expo CLI (command line tool)
             `npm install -g expo-cli`
```
Comment (HermesCui) [inline Topics/Tech_Stacks/ReactNative.md:22]: The global package expo-cli has to be deprecated

File: Topics/Tech_Stacks/ReactNative.md [hunk 3]
```diff
@@ -1,30 +1,51 @@
...
          2. Initialize expo
-            `expo init your_app_name`
+            `expo init <your_app_name>`
             #Note: you will be able to choose "blank", "blank (TypesScript)", "tabs (TypeScript)", or "minimal". I personally prefer blank because often times I need to delete files I don't want.
          3. Start expo
-            `cd your_app_name`
-            `expo start`
+            `cd <your_app_name>`
+            `npx expo start`
```
Comment (HermesCui) [inline Topics/Tech_Stacks/ReactNative.md:28]: When you change the command line, could you explain the difference?

**Review by HermesCui — COMMENTED** [review]
1. Add more details for the explanation of React Native.
2. When you change command, you need to explain the reason.

File: Topics/Tech_Stacks/ReactNative.md (hunk 1 above)
Comment (zhen-s) [reply Topics/Tech_Stacks/ReactNative.md:11]: Thanks!

File: Topics/Tech_Stacks/ReactNative.md (hunk 2 above)
Comment (zhen-s) [reply Topics/Tech_Stacks/ReactNative.md:22]: Fixed

**Comment by HermesCui** [comment]
Add some notes to explain what the different are between the original command and yours.

**Review by HermesCui — APPROVED** [review]

**Pull request merged**